include/circt/Dialect/SV/Visitors.h

```diff
@@ -66,6 +68,11 @@ class Visitor {
   HANDLE(FatalOp, Unhandled);
   HANDLE(FinishOp, Unhandled);
 
+  // Verification statements.
```

Comment:
s/Verification/Interface. ?

Reply:
Yes, copy pasta. I'll go with "Type declarations.", since it may be more than just interfaces in this class of op, and we've been calling them type declarations elsewhere.